python/pfs/datamodel/guideStars.py

```diff
@@ -213,4 +231,57 @@ def empty(cls):
                    np.array([], dtype=np.float32),
                    np.array([], dtype=np.float32),
                    0.0,
-                   0)
+                   0,
+                   np.array([], dtype='a7'),
+                   )
+
+class AutoGuiderStarMask(IntFlag):
+    """
+    Represents a bitmask for guide star properties.
+
+    Attributes:
+        GAIA: Gaia DR3 catalog.
+        HSC: HSC PDR3 catalog.
+        PMRA: Proper motion RA is measured.
+        PMRA_SIG: Proper motion RA measurement is significant (SNR>5).
+        PMDEC: Proper motion Dec is measured.
+        PMDEC_SIG: Proper motion Dec measurement is significant (SNR>5).
+        PARA: Parallax is measured.
+        PARA_SIG: Parallax measurement is significant (SNR>5).
+        ASTROMETRIC: Astrometric excess noise is small (astrometric_excess_noise<1.0).
+        ASTROMETRIC_SIG: Astrometric excess noise is significant (astrometric_excess_noise_sig<2.0).
+        RUWE: Not a binary system (RUWE<1.4).
+        PHOTO_SIG: Photometric measurement is significant (SNR>5).
+        GALAXY: Is a galaxy.
+    """
+    GAIA = auto()
```

Comment:
These should probably not use `auto`:

```suggestion
    GAIA = 0x0001
```